english/light-fire/magic-world/follow/dream/fight/sword/lazarus/lazarus.md

```diff
@@ -0,0 +1,8 @@
+Having risen from the dead, now the question is do you believe you are a:
+
+1 [god](gods/god.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.